client/app/scripts/superdesk-archive/directives.js

```diff
@@ -384,6 +384,20 @@
                 }
             };
         }])
+
+        .directive('sdItemCrops', ['metadata', function(metadata) {
+            return {
+                templateUrl: 'scripts/superdesk-archive/views/item-crops.html',
+                scope: {
+                    item: '='                
+                },
+                link: function(scope, elem) {
+                    metadata.initialize().then(function() {
+                        scope.metadata = metadata.values;
```

Comment:
Since we use only crops from metadata.values it would be good to assign only crops instead of whole metadata.values.